### PR TITLE
Web API with MongoDB: Corrected instructions for MongoDB and database creation

### LINKED ISSUE #28372: MongoDB.Driver.Core  - Myabe update instriuctions to use this nuget package now?
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 5240866d-d0a1-f0dd-7b2c-203e1d69bc4b
* Version Independent ID: 83a5cb3b-23bc-4f0c-cd1c-1d1ffeb1131a
* Content: [Create a web API with ASP.NET Core and MongoDB](https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-mongo-app?view=aspnetcore-7.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/first-mongo-app.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/first-mongo-app.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @wadepickett
* Microsoft Alias: **wpickett**

---
[Associated WorkItem - 63808](https://dev.azure.com/msft-skilling/Content/_workitems/edit/63808)

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes #28372 

- Corrected the version 6.0 instructions to run with the currently available version of MongoDB and MongoDB Shell.
- Clarified which commands were to be run at the OS command line and which ones were to be run on the MongoDB Shell.
- Specified the version of MongoDB required by the tutorial

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/first-mongo-app.md](https://github.com/dotnet/AspNetCore.Docs/blob/02476ba5831ebb48dcbd3bc90c38245b2b00c3f1/aspnetcore/tutorials/first-mongo-app.md) | [aspnetcore/tutorials/first-mongo-app](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/first-mongo-app?branch=pr-en-us-28781) |

### CODE REVIEW

File: aspnetcore/tutorials/first-mongo-app.md [hunk 1]
```diff
@@ -47,24 +47,27 @@ In this tutorial, you learn how to:
...
-
-Use the previously installed MongoDB Shell in the following steps to create a database, make collections, and store documents. For more information on MongoDB Shell commands, see [`mongosh`](https://docs.mongodb.com/mongodb-shell/run-commands/).
+Enable MongoDB and Mongo DB Shell access from anywhere on the development machine:
 
-1. Choose a directory on your development machine for storing the data. For example, *C:\\BooksData* on Windows. Create the directory if it doesn't exist. The mongo Shell doesn't create new directories.
-1. Open a command shell. Run the following command to connect to MongoDB on default port 27017. Remember to replace `<data_directory_path>` with the directory you chose in the previous step.
+1. On Windows, MongoDB is installed at *C:\\Program Files\\MongoDB* by default. Add *C:\\Program Files\\MongoDB\\Server\\\<version_number>\\bin* to the `Path` environment variable.
+1. Download and choose a directory to extract the MongoDB Shell directory to. Add the resulting path for `mongosh.exe` to the `Path` environment variable.
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-mongo-app.md:53]:
```suggestion
1. Download the MongoDB Shell and choose a directory to extract it to. Add the resulting path for `mongosh.exe` to the `Path` environment variable.
```

File: aspnetcore/tutorials/first-mongo-app.md [hunk 2]
```diff
@@ -47,24 +47,27 @@ In this tutorial, you learn how to:
...
 ## Configure MongoDB
 
-On Windows, MongoDB is installed at *C:\\Program Files\\MongoDB* by default. Add *C:\\Program Files\\MongoDB\\Server\\\<version_number>\\bin* to the `Path` environment variable. This change enables MongoDB access from anywhere on your development machine.
-
-Use the previously installed MongoDB Shell in the following steps to create a database, make collections, and store documents. For more information on MongoDB Shell commands, see [`mongosh`](https://docs.mongodb.com/mongodb-shell/run-commands/).
+Enable MongoDB and Mongo DB Shell access from anywhere on the development machine:
 
-1. Choose a directory on your development machine for storing the data. For example, *C:\\BooksData* on Windows. Create the directory if it doesn't exist. The mongo Shell doesn't create new directories.
-1. Open a command shell. Run the following command to connect to MongoDB on default port 27017. Remember to replace `<data_directory_path>` with the directory you chose in the previous step.
+1. On Windows, MongoDB is installed at *C:\\Program Files\\MongoDB* by default. Add *C:\\Program Files\\MongoDB\\Server\\\<version_number>\\bin* to the `Path` environment variable.
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-mongo-app.md:52]: Should Path be PATH?

File: aspnetcore/tutorials/first-mongo-app.md [hunk 3]
```diff
@@ -47,24 +47,27 @@ In this tutorial, you learn how to:
...
+1. Download and choose a directory to extract the MongoDB Shell directory to. Add the resulting path for `mongosh.exe` to the `Path` environment variable.
+1. Choose a directory on the development machine for storing the data. For example, *C:\\BooksData* on Windows. Create the directory if it doesn't exist. The mongo Shell doesn't create new directories.
+1. In the OS command shell (not the MongoDB Shell), use the following command to connect to MongoDB on default port 27017. Remember to replace `<data_directory_path>` with the directory chosen in the previous step.
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-mongo-app.md:55]:
```suggestion
1. In the OS command shell (not the MongoDB Shell), use the following command to connect to MongoDB on default port 27017. Replace `<data_directory_path>` with the directory chosen in the previous step.
```

**Review by tdykstra — APPROVED** [review]
LGTM, a couple of nits

File: aspnetcore/tutorials/first-mongo-app.md (hunk 2 above)
Comment (wadepickett) [reply aspnetcore/tutorials/first-mongo-app.md:52]: You are right!  It should be all caps, fixing...

File: aspnetcore/tutorials/first-mongo-app.md [hunk 4]
```diff
@@ -47,24 +47,27 @@ In this tutorial, you learn how to:
...
-
-Use the previously installed MongoDB Shell in the following steps to create a database, make collections, and store documents. For more information on MongoDB Shell commands, see [`mongosh`](https://docs.mongodb.com/mongodb-shell/run-commands/).
+Enable MongoDB and Mongo DB Shell access from anywhere on the development machine:
 
-1. Choose a directory on your development machine for storing the data. For example, *C:\\BooksData* on Windows. Create the directory if it doesn't exist. The mongo Shell doesn't create new directories.
-1. Open a command shell. Run the following command to connect to MongoDB on default port 27017. Remember to replace `<data_directory_path>` with the directory you chose in the previous step.
+1. On Windows, MongoDB is installed at *C:\\Program Files\\MongoDB* by default. Add *C:\\Program Files\\MongoDB\\Server\\\<version_number>\\bin* to the `PATH` environment variable.
+1. Download the MongoDB Shell and choose a directory to extract it to. Add the resulting path for `mongosh.exe` to the `Path` environment variable.
```
Comment (wadepickett) [inline aspnetcore/tutorials/first-mongo-app.md:53]:
```suggestion
1. Download the MongoDB Shell and choose a directory to extract it to. Add the resulting path for `mongosh.exe` to the `PATH` environment variable.
```

**Comment by wadepickett** [comment]
Some build issue not related to this doc is going on, closing and reopening to force new build.

**Pull request merged**